### PR TITLE
[Platform][nokia]: python3-smbus package add with python3 and jinja fixes 

### PR DESCRIPTION
**Pull request by carl-nokia**
**- Why I did it**
fix platform driver breakage due to python3 upgrade and fix load minigraph errors with config load_minigraph -y

**- How I did it**
added python3-smbus to the pmon docker template since the previous was python2 specific 
fixed additional "ord" python2 specific code 
fixed the jinja templates used by qos reload - the template logic required data to be parsed 

**- How to verify it**
run "show platform XXX" commands and verify output
run "sudo config load_minigraph -y" and verify configuration 
run "show interfaces XXX" and verify output 

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: device/nokia/armhf-nokia_ixs7215_52x-r0/Nokia-7215/buffers.json.j2 [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 {% set default_cable = '40m' %}
 {% set default_ports_num = 54 -%}
+{}
```
Comment (jleveque) [inline device/nokia/armhf-nokia_ixs7215_52x-r0/Nokia-7215/buffers.json.j2:5]: Why is this needed?

**Review by jleveque — CHANGES_REQUESTED** [review]
Please fill out the PR description field

File: device/nokia/armhf-nokia_ixs7215_52x-r0/Nokia-7215/buffers.json.j2 (hunk 1 above)
Comment (carl-nokia) [reply device/nokia/armhf-nokia_ixs7215_52x-r0/Nokia-7215/buffers.json.j2:5]: the rendering code required some data to operate on when processing the jinja templates used by qos reload - the template I originally submitted had some small logic but did not return data to be parsed

Comment (jleveque) [reply device/nokia/armhf-nokia_ixs7215_52x-r0/Nokia-7215/buffers.json.j2:5]: I see. Nitpicking here, but I think these braces should come after the comments below, since they should technically contain the lookup table defined in the comments.

**Comment by carl-nokia** [comment]
No problem - I see what you are saying - I changed the order

File: dockers/docker-platform-monitor/Dockerfile.j2 [hunk 2]
```diff
@@ -21,6 +21,7 @@ RUN apt-get update &&   \
         librrd-dev      \
         rrdtool         \
         python-smbus    \
+        python3-smbus   \
```
Comment (jleveque) [inline dockers/docker-platform-monitor/Dockerfile.j2:24]: Is this needed by the Nokia sonic-platform package? If so , I would prefer that you add it in the `install_requires` list of your setup.py file. This way it will get installed along with your package, and will not be installed if not needed.

Comment (carl-nokia) [reply dockers/docker-platform-monitor/Dockerfile.j2:24]: Yes, it is needed by the Nokia sonic-platform package and we see Arista using it also btw.    I attempted adding to install_requires list of setup.py file as you suggested, but it appears that mechanism is at runtime install time and did not work.  See error below:
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0xb5731730>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/python3-smbus/
Build time install solution is preferred I think  -  putting the package in our debian another way - still working on alternatives other than the current Arista method multiple debians

Comment (jleveque) [reply dockers/docker-platform-monitor/Dockerfile.j2:24]: @carl-nokia: Ah yes. The vendor's sonic-platform package gets installed in the PMon container at runtime, thus you are correct, my suggestion will not work. I am OK with this solution for now. Sorry to trouble you.

**Review by jleveque — APPROVED** [review]
